pkg/analysis/alerts_storage.go

```diff
@@ -81,13 +81,15 @@ func (s *alertsStorage) PutAlert(alert entities.Alert) (needSendAlert bool) {
 		old     = s.internalStorage[alertID]
 		repeats = old.repeats + 1
 	)
-
-	s.logger.Info("An alert was put into storage",
-		zap.Stringer("alert", alert),
-		zap.Int("backoffThreshold", old.backoffThreshold),
-		zap.Int("repeats", repeats),
-		zap.Bool("confirmed", old.confirmed),
-	)
+	defer func() {
+		info := s.internalStorage[alertID]
```

Comment:
why do you log an old alert with the same ID instead of the new one?

Reply:
I log exactly a new one because of `defer`.